review/checkbox.html

```diff
@@ -74,7 +74,7 @@ <h2>Test 1: Navigate to an unchecked checkbox in reading mode</h2>
       <ul>
         <li>Mode: reading</i>
         <li>Applies to: jaws, nvda</li>
-        <li>Lasted edited: Wed May 13 19:00:23 2020 -0500
```

Comment:
I hope this is not updated for every test file every PR because that would be pretty obnoxious..

edit: looks like it's not, according to the code